staging/src/k8s.io/apiserver/pkg/util/webhook/authentication.go

```diff
@@ -56,8 +75,20 @@ func NewDefaultAuthenticationInfoResolverWrapper(
 				if err != nil {
 					return nil, err
 				}
-				if proxyTransport != nil && proxyTransport.DialContext != nil {
-					ret.Dial = proxyTransport.DialContext
+
+				if egressLookup != nil {
+					networkContext := egressselector.Cluster.AsNetworkContext()
```

Comment:
And this choice makes sense to me for services.

Reply:
Agreed. A little uncomfortable that our only authentication webhook options are running on the Kube API Server or running in the cluster. However that does not appear to be a new restriction.